src/types/dataModel/eventVolunteer.ts

```diff
@@ -9,10 +9,7 @@ const zEventVolunteer = z.object({
   checkOutTime: z.date().optional(),
 });
 
-export const zCreateEventVolunteerRequest = zEventVolunteer.extend({
-  volunteer: zVolunteer,
-  event: zEvent,
-});
+export const zCreateEventVolunteerRequest = zEventVolunteer.extend({});
```

Comment:
don't edit the already existing zCreateEventVolunteerRequest. Make a new object called zCheckInVolunteerRequest.
This object needs to contain an eventVolunteerId type: string and a time type: Date.